apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictions/cell-population-predictions.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="container">
+  <h1>Cell Population Predictions</h1>
+  <p class="info-text">
+    This table lists all cell types predicted for the selected extraction site per tool. Note that different annotation
+    tools predict different cell types and different percentages for the very same cell type. Hover over column headers
+    for details.
+  </p>
+
+  <div class="table-container">
+    <hra-workflow-card title="Cell Population Predictions">
+      <hra-workflow-card-actions>
+        <mat-icon aria-hidden="false" aria-label="more icon" class="more-icon" fontIcon="more_vert"></mat-icon>
+      </hra-workflow-card-actions>
+      <hra-workflow-card-extra>
+        <hra-delete-file-button [fileName]="'test.csv'"></hra-delete-file-button>
```

Comment:
Get the file name from an input. @LibbyUX is currently spec'ing out what will happen when the delete icon is clicked